Docker_Windows.md

```diff
@@ -26,31 +26,33 @@
 12. Inside the downloaded folder, copy `.env.example` and name the copy `.env`
 13. Open `.env` with notepad by right-clicking it and selecting _Open With_ -> _More apps_ -> _Notepad_.
 14. Change the text after `CLIENT_PATH=` to the location of your client. The folder you are pointing to must contain a folder called `client` which should contain the client files. Use `/` instead of `\`. For example, if you have `legouniverse.exe` at `C:\Users\someone\Downloads\LU\client\legouniverse.exe`, enter `C:/Users/someone/Downloads/LU`
```

Comment:
@TheNoim, is the recommendataion here to use `/` actually required? The [Docker documentation](https://docs.docker.com/compose/env-file/) does not mention any special handling for `\` and using a `\\wsl$\Ubuntu-...` path seems to work just fine. For `\\wsl$` it looks like using forward slashes does not even work (though for regular paths such as `C:/Users/someone/...` it does).

Reply:
That was just me assuming that slashes are required because of Linux. I didn't try backslashes.

Reply:
Should this be removed then from the guide?